packages/myst-roles/src/reference.ts

```diff
@@ -5,7 +5,7 @@ const REF_PATTERN = /^(.+?)<([^<>]+)>$/; // e.g. 'Labeled Reference <ref>'
 
 export const refRole: RoleSpec = {
   name: 'ref',
-  alias: ['eq', 'numref'],
+  alias: ['eq', 'numref', 'prf:ref'],
```

Comment:
I have just added an alias for the `ref` role, otherwise all cross-referencing all ready works.